conf/application.conf

```diff
@@ -1,49 +1,124 @@
 # This is the main configuration file for the application.
 # ~~~~~
 
-# Secret key
+
+# PRODUCTS
 # ~~~~~
-# The secret key is used to secure cryptographics functions.
-# If you deploy your application to several instances be sure to use the same key!
-play.crypto.secret="changeme"
-play.crypto.secret=${?APPLICATION_SECRET}
 
-# Injection configuration
+# Product attribute list that is displayed for a specific product variant (this does not allow to select different
```

Comment:
You are not displaying the product attribute field name, but the product attribute (which consists of name and value). You have to enter the product attribute keys in this configuration, but that's explained later.